osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -7,6 +6,42 @@
 from osmaxx.excerptexport import models
 
 
+def module_converter_configuration(name, export_formats, export_options):
+    """
+    :param export_formats example:
+        {
+            'txt': {
+                'name': 'Text',
+                'file_extension': 'txt',
+                'mime_type': 'text/plain'
+            },
+            'markdown': {
+                'name': 'Markdown',
+                'file_extension': 'md',
+                'mime_type': 'text/markdown'
+            }
+        }
+    :param export_options example:
+        {
+            'image_resolution': {
+                'label': 'Resolution',
+                'type': 'number',
+                'default': '500'
+            },
+            'quality': {
+                'label': 'Quality',
+                'type': 'number',
+                'default': '10'
+            }
+        }
+    """
+    return {
```

Comment:
does this warrant a function? Or is this specifically an addition needed to be overridden during testing?